tests/recordRequests.js

```diff
@@ -1,19 +1,44 @@
 "use strict";
-const fs = require(`node:fs`);
 const path = require(`node:path`);
-const v8 = require(`node:v8`);
+const crypto = require(`node:crypto`);
+const SQLite3 = require(`better-sqlite3`);
+
+const db = new SQLite3(path.join(__dirname, `nock/nocks.db`));
```

Comment:
nit: the folder seems unnecessary if we put everything in a single file.
```suggestion
const db = new SQLite3(path.join(__dirname, `nocks.db`));
```